staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/meta.go

```diff
@@ -214,3 +216,14 @@ func (meta *ObjectMeta) GetClusterName() string {
 func (meta *ObjectMeta) SetClusterName(clusterName string) {
 	meta.ClusterName = clusterName
 }
+
+func (meta *ObjectMeta) GetLastApplied() map[string]string {
+	return meta.LastApplied
+}
+
+func (meta *ObjectMeta) SetLastApplied(lastApplied map[string]string) {
+	meta.LastApplied = make(map[string]string)
```

Comment:
tiny nit: since you know the exact size with certainty, a `, len(lastApplied)` will make this take fewer allocations.